automation/submission-perf-inc.sh

```diff
@@ -244,16 +255,33 @@ monitorSubmission() {
 
     getAccessToken "$user"
 
-    # curl -s -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/submissions/queueStatus" | jq -r '"\(now),\(.workflowCountsByStatus.Queued),\(.workflowCountsByStatus.Running),\(.workflowCountsByStatus.Submitted)"' | tee -a workflow-progress-$BUILD_NUMBER.csv
-    submissionDetails=$(curl \
-        -X GET \
-        --header 'Accept: application/json' \
-        --header "Authorization: Bearer ${ACCESS_TOKEN}" \
-        "https://api.firecloud.org/api/workspaces/$namespace/$name/submissions/$submissionId")
+    printf "\nFetching status for submission ID '%s':" "${submissionId}"
 
-    submissionStatus=$(jq -r '.status' <<< "${submissionDetails}" )
-    workflowsStatus=$(jq -r '.workflows[] | .status' <<< "${submissionDetails}")
-    workflowFailures=$(jq -r '[.workflows[] | select(.status == "Failed")] | length' <<< "${submissionDetails}")
+    # curl -s -X GET --header 'Accept: application/json' --header "Authorization: Bearer $ACCESS_TOKEN" "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/submissions/queueStatus" | jq -r '"\(now),\(.workflowCountsByStatus.Queued),\(.workflowCountsByStatus.Running),\(.workflowCountsByStatus.Submitted)"' | tee -a workflow-progress-$BUILD_NUMBER.csv
+    submissionStatus=$(
+        curl \
+            -X GET \
+            --header 'Accept: application/json' \
+            --header "Authorization: Bearer ${ACCESS_TOKEN}" \
+            "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/${namespace}/${name}/submissions/${submissionId}" \
+        | jq -r '.status'
+    )
+    workflowsStatus=$(
```

Comment:
This seems like a backwards step? Instead of fetching this value once and then interpreting it locally, we're now fetching it multiple times?

Reply:
Yes. I am not sure what is going on, but getting details once and then interpreting it locally was causing the tests to fail with `jq: error (at <stdin>:8): Cannot iterate over null (null)` ([#52](https://fc-jenkins.dsp-techops.broadinstitute.org/job/alpha-impossible-to-read-archived-metadata-from-within-workflow-check/52/console), [#1199](https://fc-jenkins.dsp-techops.broadinstitute.org/job/PerformanceTest-against-Alpha/1199/console)). Not sure why it was getting workflows[] as empty.  But reverting it back to this worked.

Reply:
Interesting! 👍  for now, and maybe I'll test locally to see what's going wrong!